packages/widget/src/components/address-input/address-input.ts

```diff
@@ -0,0 +1,32 @@
+import { LitElement, html } from 'lit';
+import type { HTMLTemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { styles } from './styles';
+
+@customElement('address-input')
+export default class AddressInput extends LitElement {
+  static styles = styles;
+  @property({
+    type: String
+  })
+  addressToTransfer?: string;
+
+  @property()
+  handleAddress?: (e: Event) => void;
```

Comment:
```suggestion
  handleAddress?: (address: string) => void;
```
make sure component parses and validates before passing address to callback